jsoc/gsoc/topopt.md

```diff
@@ -4,6 +4,16 @@
 
 The following is a tentative list of projects in topology optimisation that you could be working on in the coming Julia Season of Contributions or Google Summer of Code. If you are interested in exploring any of these topics or if you have other interests related to topology optimisation, please reach out to the main mentor [Mohamed Tarek](https://github.com/mohamed82008) via email.
 
+## Testing and benchmarking of TopOpt.jl
+
+**Project difficulty**: Easy to Medium
+
+**Work load**: 175 or 350 hours
```

Comment:
Hey @mohamed82008 , only one small concern here. I would highly advise making the executive decision of choosing either only Easy or Medium. If you feel that a project could potentially be difficult, I'd suggest just going with Medium.

With regards to the workload, I'd also suggest just stating only one time interval or another.

Reply:
done